README.md

```diff
@@ -78,6 +78,21 @@ git branch -D us-24
 
 and thats it, now `git fetch` should not pull the branches anymore
 
+## Partial Clone (Shallow Clone)
+To further reduce the size of your local repository, you can also utilize the `--depth` parameter with the `git clone` command.
+This creates a shallow clone, fetching only the most recent commits up to the specified depth, thereby ignoring the entire history that you might not need.
+
+For instance, to clone only the latest commit (depth of 1), you would use the following command:
+
+```
+git clone -b w1-24 --depth 1 https://github.com/StefanMaron/MSDyn365BC.Sandbox.Code.History
+```
+
+> [!TIP]
+> Using `--depth` implies `--single-branch` unless `--no-single-branch` is given to fetch the histories near the tips of all branches.
+
+Later, you can deepen your clone with `git fetch --deepen` or convert it to a complete clone using `git fetch --unshallow`.
```

Comment:
If I understand the `--deepen` parameter correctly its used by specifying the number of commits to ADD, right?
![image](https://github.com/StefanMaron/MSDyn365BC.Sandbox.Code.History/assets/24838311/7a4591b8-0873-4898-8edf-88b17e3265eb)

So if my initial `--depth` was 2 and I run fetch with `--deepen 3` my total depth would end up beeing 5 and not 3, correct?

Reply:
Another question:
If I specify the `depth` on clone, will all branches I might later add also have the same depth?
And what if new commit are added, will older one be dropped, or will this increase my depth? :D

Reply:
> If I understand the `--deepen` parameter correctly its used by specifying the number of commits to ADD, right?> 
> So if my initial `--depth` was 2 and I run fetch with `--deepen 3` my total depth would end up beeing 5 and not 3, correct?

This is perfectly correct 👍🏻
(And apparently --deepen doesn't work on our internal Azure DevOps. I don't know if it's due to a setting or if it's just not supported)

```
fatal: Server does not support --deepen
fatal: the remote end hung up unexpectedly
```

> If I specify the depth on clone, will all branches I might later add also have the same depth?

No, there is no reference to clone depth in the config file. The new branch will be `fetch` from its origin, except if you use `git fetch --depth X` (be careful with the auto-fetch of certain software)

> And what if new commit are added, will older one be dropped, or will this increase my depth? :D
As far as I know, no dropping of old commits, new ones are added.
(limit commits in case of shallow clone are stored in .git/shallow)